pkg/api/util.go

```diff
@@ -225,6 +225,10 @@ func (job *Job) GetPriorityClassName() string {
 	return podSpec.PriorityClassName
 }
 
+func (job *Job) GetScheduledAtPriority() (int32, bool) {
+	return -1, false
```

Comment:
I've avoided implementing home-away preemption for the old scheduler, because it's on its way out.